.github/workflows/build_packages.yaml

```diff
@@ -218,7 +218,7 @@ jobs:
         os:
           - ubuntu20.04
           - ubuntu18.04
-          - ubuntu16.04
```

Comment:
I have tested that quic can build on 16.04

Reply:
Cool，let's keep it